docs/design/coreclr/jit/WebAssembly overview for JIT.md

```diff
@@ -0,0 +1,162 @@
+# WebAssembly overview for JIT
+
+- [WebAssembly overview for JIT](#webassembly-overview-for-jit)
+  - [Introduction](#introduction)
+  - [Origins of WebAssembly](#origins-of-webassembly)
+  - [WebAssembly priorities in detail](#webassembly-priorities-in-detail)
+    - [Consistent performance](#consistent-performance)
+    - [Fast, efficient startup](#fast-efficient-startup)
+    - [Efficient, robust sandboxing](#efficient-robust-sandboxing)
+    - [Portability](#portability)
+  - [Key WebAssembly concepts \& topics](#key-webassembly-concepts--topics)
+    - [Modules](#modules)
+    - [Memories](#memories)
+    - [Global Variables](#global-variables)
+    - [Types and Signatures](#types-and-signatures)
+    - [Control Flow](#control-flow)
+    - [Function Pointers](#function-pointers)
+    - [Traps](#traps)
+    - [Shadow Stack](#shadow-stack)
```

Comment:
Nit: the term "shadow stack" is quite overloaded in WASM, I've seen it used to refer to both the linear memory stack and the WASM-VM-level stack (as used here). It may be best to use something less ambiguous (e. g. just "WASM stack"?).

Reply:
Seems LGTM.